pkg/artifactory/util_test.go

```diff
@@ -2,13 +2,15 @@ package artifactory
 
 import (
 	"fmt"
-	"gopkg.in/yaml.v2"
+	"io/ioutil"
 	"math"
 	"net/http"
 	"reflect"
 	"strings"
 	"testing"
 
+	"gopkg.in/yaml.v2"
```

Comment:
Please sort the import statements

Reply:
```
import (
	"fmt"
	"io/ioutil"
	"math"
	"net/http"
	"reflect"
	"strings"
	"testing"

	"gopkg.in/yaml.v2"

	"github.com/go-resty/resty/v2"
	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
)
```
This is how it looks like now. As per go fmt it is soreted.

Reply:
Run `make fmt` please. That uses `goimports` which has a different set of sorting rules.

Reply:
after running `make fmt` on my workspace, following files are getting modified. `util_test.go` is not in the list

```
modified:   pkg/artifactory/datasource_artifactory_file.go
        modified:   pkg/artifactory/resource_artifactory_anonymous_user.go
        modified:   pkg/artifactory/resource_artifactory_anonymous_user_test.go
        modified:   pkg/artifactory/resource_artifactory_backup.go
        modified:   pkg/artifactory/resource_artifactory_managed_user_test.go
        modified:   pkg/artifactory/resource_artifactory_pull_replication.go
        modified:   pkg/artifactory/resource_artifactory_replication_config.go
        modified:   pkg/artifactory/resource_artifactory_single_replication_config.go
        modified:   pkg/artifactory/resource_artifactory_single_replication_config_test.go
        modified:   pkg/artifactory/resource_artifactory_user.go
        modified:   pkg/artifactory/resource_artifactory_virtual_debian_repository.go
        modified:   pkg/artifactory/resource_artifactory_webhook.go
        modified:   pkg/artifactory/resource_artifactory_webhook_base.go
        modified:   pkg/artifactory/user.go
```

Reply:
Ok, the other import statements in the code have `gopkg.in/yaml.v2` in after `github.com` for obvious reason. Please make this consistent.